docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -874,18 +871,20 @@ id: download your interview
 comment: | 
 event: show_interview
 question: |
-  All done: Download your interview
+  Congratulations, you have completed your draft interview!
 subquestion: |
-
-  ### Please download the generated interview below
-  
-  Download the YAML file only:  
-  [:download: Download YAML](${interview_download.url_for(attachment=True)})
   
-  Download the interview as a Docassemble package:  
-  [:download: Download Package](${interview_package_download.url_for(attachment=True)})
+  #### Download the interview as a Docassemble package  
+  Here is your package to download: ${ interview_package_download } 
+
+  After download,<br> 
+  1. Go to your **Playground** to add a new project.<br>
+  2. Open the new project's **Packages** folder.<br>
+  3. Use the third icon **Upload** to install this package.<br>
+  4. Go back to Playground, delete test.yml file, now you can see your interview file.
   
-  ### Preview of Docassemble interview file
+  _**You can preview your interview file before you download, or copy its content if you don't want to download the whole package.**_  
```

Comment:
My intention is to simplify the screen content and accentuate actions needed, so instead of keeping 2 download links, I removed one and added download instructions. This should work for the majority users.

I would love to just say `preview` the single yml file. However, I remember when I did that in my first version (using playground), one comment was that some students may go back to rerun the Weaver and only want the yml file, so as a compromise, I added the words about copying the yml content. 

We can revise the language or ask group feedback on Friday, if that's helpful.

Reply:
Alternatively I can bring back the yml download link with an explanation to its use case, otherwise new users who didn't get our training could be confused as to why there are two download links, which was part of my original concern. 

I guess what really bothers me about this screen is the preview section, because its location seems odd. We could either move it to its own screen as we typically do, or at least move it to before the download stuff, that would improve the flow. What are your thoughts?